internal/downloader/downloader.go

```diff
@@ -62,13 +74,19 @@ func Download(ctx context.Context, destDir string, sourceUrl string, showMsg boo
 	if d, ok := ctx.Value(downloadImplKey).(downloadImpl); ok {
 		err = d.Download(ctx, destDir, []string{sourceUrl})
 	} else {
-		// conftest's Download function leverages oras under the hood to fetch from OCI. It uses the
-		// global oras client and sets the user agent to "conftest". This is not a thread safe
-		// operation. Here we get around this limitation by ensuring a single download happens at a
-		// time.
-		dlMutex.Lock()
-		err = downloader.Download(ctx, destDir, []string{sourceUrl})
-		dlMutex.Unlock()
+
+		// similar as above with respect to USEGOGATHER
+		if os.Getenv("USEGOGATHER") == "1" {
+			_, err = gatherFunc(ctx, sourceUrl, destDir)
```

Comment:
The commentary below says conftest download is not thread safe, and we need the lock/unlock. Are we thinking that gatherFunc *is* thread safe here?

Reply:
gatherFunc _should_ be thread safe here.